proof/crefine/RISCV64/Detype_C.thy

```diff
@@ -277,7 +275,8 @@ lemma cmap_relation_disjoint:
   and     ht: "s' \<Turnstile>\<^sub>c x"
   and     tp: "\<And>ko v. proj ko = Some v \<Longrightarrow> koType TYPE('a) = koTypeOf ko"
   and     xv: "x \<notin> Ptr ` {ptr..+2 ^ bits}"
-  and    sof: "size_td (typ_info_t TYPE('b)) \<le> 2 ^ objBits (undefined :: 'a)"
+  and    sof: "\<And>ko v. proj ko = Some v \<and> koType TYPE('a) = koTypeOf ko \<Longrightarrow>
+                      size_td (typ_info_t TYPE('b)) \<le> 2 ^ objBitsKO ko"
```

Comment:
Very minor, but considering the `tp` assumption above is the second conjunct here needed?

Reply:
I originally wrote this as `!!v :: 'a. size_td (type_info_t TYPE('b)) \<le> objBits v`, but had a hard time wrestling with `objBits` and `objBitsT` and `objBitsKO`, or whatever the variations are. I will see if this second conjunct can be removed but I'm not overly hopeful that it'll be easy or nice